File: Java/sibice.java

//  https://open.kattis.com/problems/sibice
import java.util.Scanner;

public class sibice {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int matches = sc.nextInt();
        int width = sc.nextInt();
        int height = sc.nextInt();
        while (matches-- > 0)
            System.out.println(sc.nextInt() <= Math.sqrt(Math.pow(width, 2) + Math.pow(height, 2)) ? "DA" : "NE");
        sc.close();
    }
}

File: C#/sibice.cs
using System;

public class sibice {
    public static void Main (string[] args) {
        string[] nums = Console.ReadLine ().Split (' ');
        int cases = int.Parse (nums[0]);
        double hypot = Math.Sqrt (Math.Pow (int.Parse (nums[1]), 2) + Math.Pow (int.Parse (nums[2]), 2));
        while (cases-- > 0)
            Console.WriteLine (int.Parse (Console.ReadLine ()) <= hypot ? "DA" : "NE");
    }
}